benchmark/scripts/Benchmark_Driver

```diff
@@ -118,28 +118,23 @@ def instrument_test(driver_path, test, num_samples):
     return avg_test_output
 
 
-BENCHMARK_OUTPUT_RE = re.compile('([^,]+),')
-
-
 def get_tests(driver_path, args):
     """Return a list of available performance tests"""
-    driver = ([driver_path, '--list'])
+    driver = ([driver_path, '--list', '--delim=\t'])
     if args.benchmarks or args.filters:
-        driver.append('--run-all')
-    tests = []
-    for l in subprocess.check_output(driver).split("\n")[1:]:
-        m = BENCHMARK_OUTPUT_RE.match(l)
-        if m is None:
-            continue
-        tests.append(m.group(1))
+        driver.append('--skip-tags=')
+    indices, names = zip(*[
```

Comment:
I always thought `zip` returns a single list. How is that supposed to work?

Reply:
I learned it over at [Stack Overflow](https://stackoverflow.com/questions/19339/transpose-unzip-function-inverse-of-zip. Adding the star before iterable will turn the zip around and do unzip. It says so right in the [official documentation](https://docs.python.org/2.7/library/functions.html#zip), too:
> zip() in conjunction with the * operator can be used to unzip a list

Reply:
I see now.. It's a little too clever. How about we at least extract is into a separate variable?

```python
indexNamePairs = [line.split('\t')[:2] for line in subprocess.check_output(driver).split('\n')[1:-1]]
indices, names = zip(*indexNamePairs)
```

Reply:
Wow. I have to say I skimmed over this line without parsing it. If python doesn't have an obvious syntax, it'd be nice to comment "# this unzips the array of pairs into two arrays"